lib/http.js

```diff
@@ -40,10 +39,7 @@ exports.get = function(options, cb) {
 exports._connectionListener = server._connectionListener;
 const Server = exports.Server = server.Server;
 
-exports.createServer = function(requestListener) {
-  return new Server(requestListener);
-};
-
+exports.createServer = requestListener => new Server(requestListener);
```

Comment:
_Please_ can we go for always-parens around the arguments when we do these? I find the terseness without them make them harder to visually parse when reading code. I think I might have @cjihrig on my side on this at least.